src/containers/profile/notifications.svelte

```diff
@@ -1 +1,121 @@
-<!--TODO: notification container of profile page-->
+<script>
+  import Button from 'ui/button.svelte';
+  import RadioGroup from 'ui/radio-group.svelte';
+
+  const items = [
+    {
+      id: 1,
+      label: 'Push',
+    },
+    {
+      id: 2,
+      label: 'Off',
+    },
+    {
+      id: 3,
+      label: 'e-mail',
+    },
+  ];
+
+  const selected = {
+    all: {},
+    store: {},
+    volunteering: {},
+    projs: {},
+  };
+
+  const changeAll = (e) => {
+    const item = e.detail;
+
+    selected.all = item;
+    selected.store = item;
+    selected.volunteering = item;
+    selected.projs = item;
```

Comment:
Would be better to use a `for .. in` loop, so that it wouldn't have to be manually changed in case more items are added in the future